### ISSUE TITLE
Ignore relative/local links?

### ISSUE THREAD

**Issue by hugovk** [tcort/markdown-link-check#10]
Would it make sense to ignore relative and local links? And perhaps bitcoin links too?

```
[✖] /bots/
[✖] #past-sponsors
[✖] #page-content
[✓] https://twitter.com/algorithm_club
[✖] /resources/libraries-frameworks/#language
[✖] bitcoin:18WDyJFShLcLVCRyiEm3MijbCsBjcxG4mH?amount=0.025&amp;label=Botwiki
```

**Comment by slythfox** [comment]
While relative link support was added via issue #2, it wasn't working with local files. I wonder if this was what you are running into. I personally want relative links to be checked.

**Comment by tcort** [comment]
There are a few issues here and they're are all sort of related... relative links, local relative links, links with different schemes (e.g. `bitcoin:`) and hash links (e.g. `#page-content`). I've got some ideas, which I wrote on pull request #13, that I'll implement.

**Comment by PeterDaveHello** [comment]
👍

**Comment by dialex** [comment]
I have the exact same problem. I tried the feature implemented in #13 but that didn't work for me. I need a command that works both locally and on CI (Travis). It needs to check recursively all `md` files inside a folder.

How can I do that?

**Comment by timmkrause** [comment]
Is this solved by #41 / with version 3.5.0?
